### ISSUE TITLE
Pagination not working for session.contenthost.search

### ISSUE THREAD

**Issue by swadeley** [SatelliteQE/airgun#570]
Hello

session.contenthost.search(f"os = {module_host_template.operatingsystem.name}")
 has stopped searching more than one page in Content Host view.

https://github.com/SatelliteQE/robottelo/blob/3dbd842e6fbb6ea69b5f4d9ec5f13d0e09398ddc/tests/foreman/ui/test_contenthost.py#L1285

[ Add test for C-Host pagination #7738 ](https://github.com/SatelliteQE/robottelo/pull/7738)

**Comment by swadeley** [comment]
Hello @tpapaioa 

I think this commit[1] causes the problem:

commit 6a75c0aefe1a55bea5efecd559d24c555c450d7b
Author: Tasos Papaioannou <tpapaioa@redhat.com>
Date:   Mon Nov 30 09:31:37 2020 -0500

    Fix Pagination ROOT locator issue (#536)

If I get reset before that commit, and then add back this one:

airgun]$ git cherry-pick c5cde330b901ca2a2acabec9eaeaab9be29e3123
Auto-merging airgun/widgets.py
[master 4824857] Add ROOT for Search widget (#545)
 Author: tpapaioa <tpapaioa@redhat.com>
 Date: Wed Jan 6 22:42:17 2021 -0500
 1 file changed, 1 insertion(+)

Then test passes again.

[1] https://github.com/SatelliteQE/airgun/pull/541

**Comment by tpapaioa** [comment]
The `SatTable` class has a `Pagination` subwidget, which is used by `SatTable.read()` if displayed:

```
class SatTable(Table):
[...]
    pagination = Pagination()
[...]
    def read(self):
[...]
        if self.pagination.is_displayed:
            return self._read_all()
```

Because the paginator in the UI is not actually a sub-element of the table element, the change to `Pagination.ROOT` from an absolute to a relative xpath locator broke this use case where the paginator is included in the table widget. A different `ROOT` for these nested paginators would probably be the easiest fix. A better longer term would probably be to refactor this search / table / paginator combo as a nested view.
